src/main/java/pl/stormit/eduquiz/statistic/quizstatistic/QuizStatisticFacade.java

```diff
@@ -16,5 +17,5 @@ public interface QuizStatisticFacade {
 
     QuizStatisticDto addStatisticToDB(Game game, int score);
 
-    
+    Map<String, Long> getDurationForEachQuizzes(boolean best);
```

Comment:
Map name could be "getDurationForEachQuiz" instead of plural form "...ForEachQuizzes". Not a big deal, but it would look more correct grammatically